src/test/java/catobot/CatobotTest.java

```diff
@@ -0,0 +1,87 @@
+package catobot;
+
+import catobot.command.Command;
+import catobot.command.Parser;
+import catobot.exception.*;
+import catobot.item.Task;
+import catobot.item.TaskList;
+import catobot.item.Todo;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CatobotTest {
+    private static final String filePath = "./data/Catobot.txt";
+
+    @Test
+    public void taskList_addTodo_todoAddedToTaskList() {
+        String description = "todo demo_description";
+        String expected = "";
+        String actual = "";
+        try {
+            Task todo = Todo.of(description);
+            expected = todo.toString();
+            actual = new TaskList().add(todo).split("\n")[1].trim();
+        } catch (BotException e) {
+            actual = e.getMessage();
+        }
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parser_invalidCommand_invalidCommandExceptionThrown() {
+        String expected = new InvalidCommandException().getMessage();
+        String actual = "";
+        try {
+            Parser.parse("helloworld 1");
+        } catch (BotException e) {
+            actual = e.getMessage();
+        }
+        assertEquals(expected, actual);
+        assertEquals(expected, actual);
```

Comment:
There seems to be a repetition here?

Reply:
Yes you are right, thanks for pointing it out.